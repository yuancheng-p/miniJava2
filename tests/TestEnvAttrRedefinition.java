package tutu.titi.toto;

class I {}

public class A {
    Int a1 = 1;
    int[] a = { 3,4,5 };
    Bool a2 = false;
    String a6 = "jsdjkndjk\njdsdjkf";
    Book a1;

}


