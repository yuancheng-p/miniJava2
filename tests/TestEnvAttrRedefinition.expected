successfull parsing
package tutu.titi.toto
class I extends Object {
}

public class A extends Object {
  Int a1 = 1;
  int[] a = {3,4,5};
  Bool a2 = false;
  String a6 = "jsdjkndjk\njdsdjkf";
  Book a1;
}

start typing
